crates/provider-sdk/src/otel.rs

```diff
@@ -69,6 +69,42 @@ macro_rules! initialize_observability {
                     "OTEL_EXPORTER_OTLP_LOGS_ENDPOINT" => {
                         otel_config.logs_endpoint = Some(v.clone())
                     }
+                    "WASMCLOUD_OBSERVABILITY_TRACE_SAMPLING" => {
```

Comment:
`OTEL_EXPORTER_OTLP_TRACES_SAMPLING`

( and think the others should be `OTEL_EXPORTER_OTLP_` prefix as well )

Reply:
Is that a standard env var? The ones that were `OTEL` prefixed before are standard ones. The println was a mistake from tab completing an AI suggestion I think 😅

Reply:
Lol it is: https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#batch-span-processor

Yaaaaaay